Seafile 2.0.1: Material Design(?) crash on Android 4.4 or older

First observed on a "Samsung T-113" Tablet.

Steps to reproduce it in the emulator:

1. Create a new AVD based on API level 17 or 19 ("Nexus 4" template)
2. start Emulator & install seadroid
3. create seadroid account
4. open a file in BrowserActivity (e.g. ".pdf") (file must not be cached)
5. the attached crash happens.

Additional notes:
- Android 5.1 or 6.0 are not affected
- Downloading a file into the cache works. And opening a cached file will work as well.

```
01-24 16:31:38.197 3481-3481/com.seafile.seadroid2 D/BrowserActivity: onPause
01-24 16:31:38.197 3481-3481/com.seafile.seadroid2 D/DefaultAppLock: onActivityPaused
01-24 16:31:38.207 3481-3481/com.seafile.seadroid2 D/Account: https://xxxx.xxxx.xxxx/ xxxxx@xxxxx.xxxx xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
01-24 16:31:38.207 3481-3481/com.seafile.seadroid2 I/AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
01-24 16:31:38.207 3481-3481/com.seafile.seadroid2 W/ResourceType: Too many attribute references, stopped at: 0x01010099
01-24 16:31:38.217 3481-3481/com.seafile.seadroid2 W/ResourceType: Too many attribute references, stopped at: 0x0101009a
01-24 16:31:38.217 3481-3481/com.seafile.seadroid2 W/ResourceType: Too many attribute references, stopped at: 0x0101009b
01-24 16:31:38.217 3481-3481/com.seafile.seadroid2 D/AndroidRuntime: Shutting down VM
01-24 16:31:38.217 3481-3481/com.seafile.seadroid2 W/dalvikvm: threadid=1: thread exiting with uncaught exception (group=0xb0d1ab20)
01-24 16:31:38.217 3481-3481/com.seafile.seadroid2 E/AndroidRuntime: FATAL EXCEPTION: main
                                                                     Process: com.seafile.seadroid2, PID: 3481
                                                                     java.lang.RuntimeException: Unable to start activity ComponentInfo{com.seafile.seadroid2/com.seafile.seadroid2.ui.activity.FileActivity}: android.view.InflateException: Binary XML file line #64: Error inflating class Button
                                                                         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2195)
                                                                         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245)
                                                                         at android.app.ActivityThread.access$800(ActivityThread.java:135)
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                         at android.os.Looper.loop(Looper.java:136)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5017)
                                                                         at java.lang.reflect.Method.invokeNative(Native Method)
                                                                         at java.lang.reflect.Method.invoke(Method.java:515)
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)
                                                                         at dalvik.system.NativeStart.main(Native Method)
                                                                      Caused by: android.view.InflateException: Binary XML file line #64: Error inflating class Button
                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:714)
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:756)
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:759)
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:759)
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:397)
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:353)
                                                                         at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:256)
                                                                         at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:109)
                                                                         at com.seafile.seadroid2.ui.activity.BaseActivity.setContentView(BaseActivity.java:71)
                                                                         at com.seafile.seadroid2.ui.activity.FileActivity.onCreate(FileActivity.java:82)
                                                                         at android.app.Activity.performCreate(Activity.java:5231)
                                                                         at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087)
                                                                         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2159)
                                                                         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245) 
                                                                         at android.app.ActivityThread.access$800(ActivityThread.java:135) 
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196) 
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                         at android.os.Looper.loop(Looper.java:136) 
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5017) 
                                                                         at java.lang.reflect.Method.invokeNative(Native Method) 
                                                                         at java.lang.reflect.Method.invoke(Method.java:515) 
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779) 
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595) 
                                                                         at dalvik.system.NativeStart.main(Native Method) 
                                                                      Caused by: java.lang.UnsupportedOperationException: Can't convert to color: type=0x2
                                                                         at android.content.res.TypedArray.getColor(TypedArray.java:327)
                                                                         at android.widget.TextView.<init>(TextView.java:677)
                                                                         at android.widget.Button.<init>(Button.java:107)
                                                                         at android.support.v7.widget.AppCompatButton.<init>(AppCompatButton.java:62)
                                                                         at android.support.v7.widget.AppCompatButton.<init>(AppCompatButton.java:58)
                                                                         at android.support.v7.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:98)
                                                                         at android.support.v7.app.AppCompatDelegateImplV7.createView(AppCompatDelegateImplV7.java:938)
                                                                         at android.support.v7.app.AppCompatDelegateImplV7.onCreateView(AppCompatDelegateImplV7.java:992)
                                                                         at android.support.v4.view.LayoutInflaterCompatHC$FactoryWrapperHC.onCreateView(LayoutInflaterCompatHC.java:44)
                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:685)
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:756) 
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:759) 
                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:759) 
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:492) 
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:397) 
                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:353) 
                                                                         at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:256) 
                                                                         at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:109) 
                                                                         at com.seafile.seadroid2.ui.activity.BaseActivity.setContentView(BaseActivity.java:71) 
                                                                         at com.seafile.seadroid2.ui.activity.FileActivity.onCreate(FileActivity.java:82) 
                                                                         at android.app.Activity.performCreate(Activity.java:5231) 
                                                                         at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1087) 
                                                                         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2159) 
                                                                         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2245) 
                                                                         at android.app.ActivityThread.access$800(ActivityThread.java:135) 
                                                                         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1196) 
                                                                         at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                         at android.os.Looper.loop(Looper.java:136) 
                                                                         at android.app.ActivityThread.main(ActivityThread.java:5017) 
                                                                         at java.lang.reflect.Method.invokeNative(Native Method) 
                                                                         at java.lang.reflect.Method.invoke(Method.java:515) 
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779) 
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595) 
                                                                         at dalvik.system.NativeStart.main(Native Method) 
01-24 16:31:39.367 3481-3481/com.seafile.seadroid2 I/Process: Sending signal. PID: 3481 SIG: 9
```

--- a/app/src/main/res/layout/file_activity.xml
+++ b/app/src/main/res/layout/file_activity.xml
@@ -69,7 +69,6 @@
                 android:layout_marginLeft="@dimen/long_btn_margin_left"
                 android:layout_marginRight="@dimen/long_btn_margin_right"
                 android:text="@string/cancel"
-                android:textColor="@color/light_black"
                 android:textSize="@dimen/long_btn_txt_size"
                 android:theme="@style/NegativeButton" />
         </LinearLayout>

--- a/app/src/main/res/values-v21/styles.xml
+++ b/app/src/main/res/values-v21/styles.xml
@@ -34,9 +34,9 @@
         <item name="android:textColor">@color/fancy_orange</item>
     </style>
 
-    <style name="NegativeButton" parent="android:style/Widget.Material.Button">
+    <style name="NegativeButton" parent="Base.Widget.AppCompat.Button">
         <item name="android:colorButtonNormal">@color/fancy_dark_gray</item>
-        <item name="colorButtonNormal">@color/fancy_dark_gray</item>
+        <item name="android:textColor">@color/light_black</item>
     </style>
 
     <style name="ActionBarPopupThemeOverlay" parent="ThemeOverlay.AppCompat.Light" />

--- a/app/src/main/res/values/styles.xml
+++ b/app/src/main/res/values/styles.xml
@@ -53,9 +53,9 @@
         <item name="android:textColor">@color/fancy_dark_black</item>
     </style>
 
-    <style name="NegativeButton" parent="android:style/Widget.Material.Button">
-        <item name="android:colorButtonNormal">@color/fancy_dark_gray</item>
+    <style name="NegativeButton" parent="Base.Widget.AppCompat.Button">
         <item name="colorButtonNormal">@color/fancy_dark_gray</item>
+        <item name="android:textColor">@color/light_black</item>
     </style>
 
     <style name="DialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert">

